Sources/Commands/SwiftPackageCollectionsTool.swift

```diff
@@ -0,0 +1,358 @@
+            let swiftVersions = optionalRow("Verified Swift Versions", version.verifiedSwiftVersions?.map { $0.rawValue }.joined(separator: ", "))
+            let license = optionalRow("License", version.license?.type.description)
+            let cves = optionalRow("CVEs", version.cves?.map { $0.identifier }.joined(separator: ", "))
+
+            return """
+                \(version.version)
+                Package Name: \(version.packageName)
+                Modules: \(modules)\(platforms)\(swiftVersions)\(license)\(cves)
+            """
+        }
+
+        mutating func run() throws {
+            guard let packageUrl = packageUrl else {
+                throw CollectionsError.missingArgument("packageUrl")
+            }
+
+            let identity = PackageReference.computeIdentity(packageURL: packageUrl)
+            let reference = PackageReference(identity: identity, path: packageUrl)
+
+            let result = try await { self.collections.getPackageMetadata(reference, profile: profileOptions.usedProfile, callback: $0) }
+
+            if let versionString = version {
+                guard let version = TSCUtility.Version(string: versionString), let result = result.package.versions.first(where: { $0.version == version }), let printedResult = printVersion(result) else {
+                    throw CollectionsError.invalidVersionString(versionString)
+                }
+
+                print("Version: \(printedResult)")
+            } else {
+                let description = optionalRow("Description", result.package.description)
+                let versions = result.package.versions.map { "\($0.version)" }.joined(separator: ", ")
+                let watchers = optionalRow("Watchers", result.package.watchersCount?.description)
+                let readme = optionalRow("Readme", result.package.readmeURL?.absoluteString)
+                let authors = optionalRow("Authors", result.package.authors?.map { $0.username }.joined(separator: ", "))
+                let latestVersion = optionalRow("--------------------------------------------------------------\nLatest Version", printVersion(result.package.latestVersion))
+
+                print("""
+                    \(description)Available Versions: \(versions)\(watchers)\(readme)\(authors)\(latestVersion)
+                """)
+            }
+        }
+    }
+}
+
+private func optionalRow(_ title: String, _ contents: String?) -> String {
+    if let contents = contents {
+        return "\n\(title): \(contents)\n"
+    } else {
+        return ""
+    }
+}
+
+private extension JSONEncoder {
+    func print<T>(_ value: T) throws where T : Encodable {
+        if #available(macOS 10.15, *) {
+            self.outputFormatting = [.prettyPrinted, .sortedKeys, .withoutEscapingSlashes]
+        }
+
+        let jsonData = try self.encode(value)
+        let jsonString = String(data: jsonData, encoding: .utf8)!
+        Swift.print(jsonString)
```

Comment:
Is this just for debugging?  Otherwise, is sending to stdout always the right thing?  JSON on stdout is often problematic since other output will interfere (even debug output emitted while developing SwiftPM itself).  If this is intended to be an output format it would be good to have a command line flag for where to put the JSON.

Reply:
Yes, this is identical to existing SwiftPM CLI features which emit JSON. If that's a concern, I would prefer to holistically address that for all commands in a separate PR than partially just for the new commands.

Reply:
Makes sense.  Eventually we should address it for all of them.  Scraping stdout is never a good idea.